ci/ci/ci.py

```diff
@@ -27,8 +27,11 @@
 from .environment import BUCKET
 from .github import Repo, FQBranch, WatchedBranch, UnwatchedBranch, MergeFailureBatch
 
-with open(os.environ.get('HAIL_CI_OAUTH_TOKEN', 'oauth-token/oauth-token'), 'r') as f:
-    oauth_token = f.read().strip()
+oauth_token = None
+HAIL_CI_OAUTH_TOKEN = os.getenv('HAIL_CI_OAUTH_TOKEN')
+if HAIL_CI_OAUTH_TOKEN:
+    with open(HAIL_CI_OAUTH_TOKEN, 'r') as f:
+        oauth_token = f.read().strip()
 
```

Comment:
What's the behaviour on the startup when initialising the `gh_aiohttp.GitHubAPI` when the oauth_token is `None`:
https://github.com/populationgenomics/hail/blob/b6f287e891b39e61e8c76092855a15c9f1d5c864/ci/ci/ci.py#L311

Reply:
The OAuth token is optional. Cloning public branches (which is what we'd do for a `dev deploy`) works fine without a token.

I've cleaned up this file a bit more to only leave the functions that are necessary for `dev deploy`.